src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -72,8 +72,9 @@ def load_arguments(self, _):
         c.argument('node_zones', zones_type, options_list='--node-zones', help='(PREVIEW) Space-separated list of availability zones where agent nodes will be placed.')
         c.argument('enable_pod_security_policy', action='store_true')
         c.argument('node_resource_group')
-        c.argument('enable_acr', is_preview=True)
-        c.argument('acr_name', is_preview=True)
```

Comment:
@yugangw-msft I removed the `is_preview=True` because its documentation is in different formats compared to other preview arguments.

Reply:
I see.
On other option is can you fix the other preview related arguments, instead of continue the inconsistency with the rest command modules?

Reply:
I'd prefer to do it in a separate PR if it's something we want to do.

Reply:
For current changes, I think keeping it in the same style as others is better.